node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/NodePrioritizer.java

```diff
@@ -187,18 +190,23 @@ void addNewDockerNodes() {
 
             if (!hostHasCapacityForWantedFlavor || conflictingCluster) continue;
 
+            log.log(LogLevel.DEBUG, "Trying to add new Docker node on " + node);
             Set<String> ipAddresses = DockerHostCapacity.findFreeIps(node, allNodes);
             if (ipAddresses.isEmpty()) continue;
             String ipAddress = ipAddresses.stream().findFirst().get();
             Optional<String> hostname = nameResolver.getHostname(ipAddress);
-            if (!hostname.isPresent()) continue;
+            if (!hostname.isPresent()) {
+                log.log(LogLevel.DEBUG, "Could not find hostname for " + ipAddress + ", skipping it");
+                continue;
+            }
             Node newNode = Node.createDockerNode("fake-" + hostname.get(),
                                                  Collections.singleton(ipAddress),
                                                  Collections.emptySet(), hostname.get(),
                                                  Optional.of(node.hostname()), getFlavor(requestedNodes),
                                                  NodeType.tenant);
             PrioritizableNode nodePri = toNodePriority(newNode, false, true);
             if (!nodePri.violatesSpares || isAllocatingForReplacement) {
+                log.log(LogLevel.DEBUG, "Adding new Docker node " + node);
```

Comment:
This should (probably) be `newNode`, `node` is the parent. Unless you wanted to say "Adding new Docker node _to_ `node`"?

Reply:
Yes, thanks, fixed.